test/end-to-end/pages/viewRecipe.js

```diff
@@ -52,6 +52,26 @@ module.exports = class ViewRecipePage extends MainPage {
     this.toolBarMoreActionList = {
       Export: 'Export',
     };
+
+    // Detail tab element
+    this.detailTabElement = `${this.tabListRootElement} li:nth-child(1) a`;
+
+    // Components tab page element
+    this.componentsTabElement = `${this.tabListRootElement} li:nth-child(2) a`;
+
+    // Compositions tab page element
+    this.compositionsTabElement = `${this.tabListRootElement} li:nth-child(3) a`;
+
+    // Selected Components tab under Components tab
+    this.tabSelectedComponents = `${this.componentsContentRootElement} pf-tabs ul[role="tablist"] li:nth-child(1) a`;
+
+    // Dependencies tab under Components tab
+    this.tabDependencies = `${this.componentsContentRootElement} pf-tabs ul[role="tablist"] li:nth-child(2) a`;
+
+    // Selected Components content
+    this.contentSelectedComponents = `${this.componentsContentRootElement} pf-tabs pf-tab
+                                      div[class="list-pf cmpsr-list-pf list-pf-stacked cmpsr-recipe__components"]`;
+   }
```

Comment:
@jgiardino, could you please remove this additional bracket. It caused lint error and test error. Thanks.